definitions/procedures/backup/config_files.rb

```diff
@@ -59,17 +73,20 @@ def config_files
         exclude_configs += feature.config_files_to_exclude
         exclude_configs += feature.config_files_exclude_for_online if @online_backup
       end
+      [configs, exclude_configs]
+    end
 
-      if feature(:foreman_proxy)
-        configs += feature(:foreman_proxy).config_files(@proxy_features)
-        exclude_configs += feature(:foreman_proxy).config_files_to_exclude(@proxy_features)
-      end
+    def foreman_proxy_configs
+      feature(:foreman_proxy).config_files(@proxy_features)
+    end
 
-      configs.compact.select { |path| Dir.glob(path).any? }
-      exclude_configs.compact.select { |path| Dir.glob(path).any? }
-      [configs, exclude_configs]
+    def foreman_proxy_exclude_configs
+      feature(:foreman_proxy).config_files_to_exclude(@proxy_features)
+    end
+
+    def deb_config_dirs
+      feature(:foreman_database).config_dirs if debian_or_ubuntu?
```

Comment:
And then this should be
```suggestion
    def config_dirs
      debian_or_ubuntu? ? feature(:foreman_database).config_dirs : []
```